CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 6.4.1
+
+### Patch Changes
+
+- 8d11e15: Improve docs and types for most options, adding `SVGElement` as a supported type of "DOM node" since it supports the `focus()` method, same as `HTMLElement`.
```

Comment:
I was getting a merge conflict appearing against the readme file so I merged with master to resolved the conflict.

Reply:
Looks like you merged from somewhere else because this change should only be on the `changeset-release/master` branch, not actual `master`. That other branch is created automatically by the Changeset bot.

Reply:
Damn, I merged `changeset-release/master` instead of `master`.

I think I saw "master" in the name and saw that it was the latest commit so I merged that instead of merging actual master branch.

I'll need to redo the merge ☹

Reply:
Shoot! I _think_ you might be able to get away with just reverting the changes to the CHANGELOG and package.json because that branch would've been based on `master`, which explains why you got the README and index.d.ts changes, but also got the extra stuff.

So I would look at what's on `changeset-release/master` and just selectively remove those changes from this PR instead of trying to figure out a way to revert the merge and redo the rebase.

Reply:
It's ok. I'll reset my branch to before the merge, make the merge again (this time against master) and cherry pick any other changes I've made. This isn't hard to do in a GUI.

I'll need to do a force-push though to get rid of the bad merge on this branch.

Reply:
This PR is branching off of `master` branch now instead of `changeset-release/master`